### PR TITLE
Translate: migrate published  v3 translate batch samples

### PR DESCRIPTION
**Pull request by munkhuushmgl**

### CODE REVIEW

**Comment by munkhuushmgl** [comment]
PTAL @nnegrey

File: translate/automl/translate_v3_batch_translate_text_with_model.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+# limitations under the License.
+
+
+# [START translate_v3_batch_translate_text_with_model]
+from google.cloud import translate
+
+
+def batch_translate_text_with_model(
+    input_uri="YOUR_INPUT_URI",
+    output_uri="YOUR_OUTPUT_URI",
```
Comment (nnegrey) [inline translate/automl/translate_v3_batch_translate_text_with_model.py]: Does these have to be gcs uris?
What type of input file? 
Is the output uri a directory?

```
input_uri="gs://YOUR_BUCKET_ID/path/to/your/file.type",
output_uri="gs://YOUR_BUCKET_ID/path/to/save/results/"
```

File: translate/automl/translate_v3_batch_translate_text_with_model.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+     Model can be AutoML or General[built-in] model. """
+
+    client = translate.TranslationServiceClient()
+
+    gcs_source = {"input_uri": input_uri}
+    location = "us-central1"
+
+    # Supported language codes: https://cloud.google.com/translate/docs/languages
+    target_lang = "ja"
+    mime_type = "text/plain"  # Can be "text/plain" or "text/html".
```
Comment (nnegrey) [inline translate/automl/translate_v3_batch_translate_text_with_model.py]: Inline these

File: translate/automl/translate_v3_batch_translate_text_with_model.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    input_configs_element = {"gcs_source": gcs_source, "mime_type": mime_type}
+    input_configs = [input_configs_element]
```
Comment (nnegrey) [inline translate/automl/translate_v3_batch_translate_text_with_model.py]: Inline this

**Comment by munkhuushmgl** [comment]
Looks like something is broken outside of translate

File: translate/automl/translate_v3_batch_translate_text_with_model.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START translate_v3_batch_translate_text_with_model]
+from google.cloud import translate
+
+
+def batch_translate_text_with_model(
+    input_uri="gs://YOUR_BUCKET_ID/path/to/your/file.type",
```
Comment (nnegrey) [inline translate/automl/translate_v3_batch_translate_text_with_model.py]: Change `.type` to whatever is supported by the API. 

Probably worth adding a link below to this page for more info about the supported file types. 
https://cloud.google.com/translate/docs/reference/rpc/google.cloud.translation.v3#google.cloud.translation.v3.InputConfig

Comment (munkhuushmgl) [reply translate/automl/translate_v3_batch_translate_text_with_model.py]: this link is a bit long for including the sample, I have added this link instead 
https://cloud.google.com/translate/docs/supported-formats

File: translate/automl/translate_v3_batch_translate_text_with_model.py [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    input_configs_element = {
+        "gcs_source": gcs_source,
+        "mime_type": "text/plain"  # Can be "text/plain" or "text/html".
+    }
+    gcs_destination = {"output_uri_prefix": output_uri}
+    output_config = {"gcs_destination": gcs_destination}
+    parent = client.location_path(project_id, location)
+
+    model_path = "projects/{}/locations/{}/models/{}".format(
+        project_id, "us-central1", model_id  # The location of AutoML model.
```
Comment (nnegrey) [inline translate/automl/translate_v3_batch_translate_text_with_model.py]: Should this `us-central1` use the location variable or should the variable be removed and just use inline across the board?

**Comment by munkhuushmgl** [comment]
> Just that one last comment, thanks!

YW, I dont have permission to merge this PR, Could you merge it for me once the tests are passed?

**Comment by czahedi** [comment]
Merging per prior LGTMs and tests passing

**Pull request merged**